src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(){
```

Comment:
This method needs header comments, without which, the client overriding this will not know how to use it.